Yoast/wordpress-seo
# Deprecate theme title hooks

## Summary


This PR can be summarized in the following changelog entry:

* N/A 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run `composer install` / `composer du`
* Apply the next filter and verify you get a deprecation warning/error/something:
```
add_action( 'init', function() {  
   apply_filters( 'woo_title', 'This is a title' ); 
} );
```
* Apply the next filter and verify you get a deprecation warning/error/something:
```
add_action( 'init', function() {  
   apply_filters( 'thematic_doctitle', 'This is a title' ); 
} );
```
* This is it!


## Quality assurance


Fixes #
